### PR TITLE
Adding Email popup back to cfgov-refresh

### PR DESCRIPTION
**Pull request by sebworks**
Adding Email popup back to cfgov-refresh

## Testing

1. Enable the flag for the appropriate route ( http://localhost:8000/owning-a-home/ )
2. Visit `http://localhost:8000/owning-a-home/`.
3. Scroll to the bottom of the page and verify that the popup appears.
4. Enter in a incorrect email address in the popup and verify the validation works.
5. Enter in a correct email address in the popup and verify the email is submitted correctly.

## Notes

- I'm certain there are better ways to get emails on file and encourage topic specific engagement. 

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] Any *change* in functionality is tested
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

File: cfgov/jinja2/v1/sublanding-page/index.html [hunk 1]
```diff
@@ -53,6 +53,10 @@
             {{ render_block.render(block, loop.index) }}
         {% endif %}
     {%- endfor %}
+
+    {% if flag_enabled('EMAIL-POPUP', request) %}
```
Comment (Scotchester) [inline cfgov/jinja2/v1/sublanding-page/index.html]: Typo: Should be `EMAIL_POPUP` (with an underscore, not a hyphen).

File: cfgov/unprocessed/css/main.less [hunk 2]
```diff
@@ -132,6 +132,7 @@
 @import (less) "organisms/sidebar-content.less";
 @import (less) "organisms/snippet-list.less";
 @import (less) "organisms/well.less";
+@import (less) "organisms/email-popup.less";
```
Comment (Scotchester) [inline cfgov/unprocessed/css/main.less]: Trivial: Please alphabetize this in the list.

(Just realized I accidentally didn't queue up the others in a review. Sorry! Going to keep not doing that to be consistent.)

File: cfgov/jinja2/v1/_includes/organisms/email-popup.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+   Sidebar popup that fades in upon scrolling on the Owning a Home sublanding
+   page at /owning-a-home/.
+
+   Creates Pop-up organism:
+
+   ========================================================================== #}
+
+<div class="o-email-popup">
+  <div class="o-email-popup_header u-clearfix">
```
[the remark this answers is not in the document]
Comment (sebworks) [reply cfgov/jinja2/v1/_includes/organisms/email-popup.html]: fixed in https://github.com/cfpb/cfgov-refresh/pull/3484/commits/4aabd3c7c07f2bc6db6befbff83f37f1c6915eb3

File: cfgov/jinja2/v1/sublanding-page/index.html (hunk 1 above)
Comment (sebworks) [reply cfgov/jinja2/v1/sublanding-page/index.html]: fixed in https://github.com/cfpb/cfgov-refresh/pull/3484/commits/4aabd3c7c07f2bc6db6befbff83f37f1c6915eb3

File: cfgov/unprocessed/css/main.less (hunk 2 above)
Comment (sebworks) [reply cfgov/unprocessed/css/main.less]: fixed in https://github.com/cfpb/cfgov-refresh/pull/3484/commits/4aabd3c7c07f2bc6db6befbff83f37f1c6915eb3

File: cfgov/jinja2/v1/_includes/organisms/email-popup.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                  enctype="application/x-www-form-urlencoded"
+                 novalidate>
+                <p>Sign up for email tips and info to help you through the process.</p>
+                <div class="m-form-field-with-button" data-qa-hook="formfield-with-button">
+                    <div class="form-group">
+                        <input id="form_18"
+                               type="email"
+                               placeholder="Enter your email address"
+                               name="email"
+                               class="m-form-field-with-button_field"
```
Comment (virginiacc) [inline cfgov/jinja2/v1/_includes/organisms/email-popup.html]: Does the email address input need the `a-text-input` class? 

screen shot 2017-10-18 at 11 15 47 am

Comment (sebworks) [reply cfgov/jinja2/v1/_includes/organisms/email-popup.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3484/commits/5fd38ec2dd69e636764b679629644d599226fbdc

**Pull request merged**